### PR TITLE
Update .gitattributes file

### PR DESCRIPTION
**Pull request by ker0x**
@TomasVotruba as suggested in #635, I opened a new PR that is sync with master

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,7 +1,16 @@
...
+/tests                  export-ignore
+/utils                  export-ignore
+/.editorconfig          export-ignore
+/.gitattributes         export-ignore
+/.gitignore             export-ignore
+/.kodiak.toml           export-ignore
+/CODE_OF_CONDUCT.md     export-ignore
+/ecs.php                export-ignore
+/phpstan.neon           export-ignore
+/phpstan-extension.neon export-ignore
```
Comment (Kocal) [inline .gitattributes:14]: To keep, otherwise we won't be able to use the PHPStan extension

File: .gitattributes [hunk 2]
```diff
@@ -1,7 +1,16 @@
...
-CODE_OF_CONDUCT.md export-ignore
-README.md          export-ignore
-docs               export-ignore
-tests              export-ignore
+* text=auto
+
+/.github                export-ignore
+/docs                   export-ignore
+/tests                  export-ignore
+/utils                  export-ignore
```
Comment (Kocal) [inline .gitattributes:6]: To keep, otherwise we won't be able to use the PHPStan extension

Comment (Kocal) [reply .gitattributes:6]: Or you can be futur-proof and use the following I guess:
```gitignore
/utils/*
!/utils/phpstan-behaviors
```

**Comment by TomasVotruba** [comment]
@ker0x Thank you. Could you process the feedback from @Kocal ?

**Comment by ker0x** [comment]
@TomasVotruba It's done 😉

**Comment by Kocal** [comment]
Thanks !)

**Comment by TomasVotruba** [comment]
Thanks :+1:

**Pull request merged**